nncf/openvino/graph/node_utils.py

```diff
@@ -468,3 +468,36 @@ def get_weighted_layer_attributes(
         }
         return LinearLayerAttributes(**kwargs)
     return GenericWeightedLayerAttributes(weight_requires_grad=False, weight_shape=attrs.get("shape", None))
+
+
+def get_activation_channel_axis(node: NNCFNode, port_id: Optional[int] = None) -> int:
+    """
+    Returns axis number of the activation tensor which correspond to it channel.
+
+    :param node: NNCFNode instance.
+    :param port_id: Specified input port id if needed.
+    :return: Channel axis number.
+    """
+    channel_axis = 1
+
+    if node.metatype == OVMatMulMetatype:
+        if port_id is None:
+            raise nncf.InternalError("port_id is required for OVMatMulMetatype with transpose.")
+        if port_id > 1:
+            raise nncf.InternalError(f"{node.metatype.name} can not take more than 2 input tensors.")
```

Comment:
I suggest to make a `get_linear_weights_layout_from_node` function in the same fashion as the `get_linear_weights_layout_from_node` function.

This should help to write tests quickly as  tests for `get_linear_weights_layout_from_node` should be similar to this one

Reply:
As you wish. Done.

Reply:
Nice! Thanks a lot!